blaze/compute/sql.py

```diff
@@ -1181,7 +1181,11 @@ def compute_up(expr, data, **kwargs):
 
 
 @dispatch(Expr, ClauseElement)
-def post_compute(_, s, **kwargs):
+def post_compute(expr, s, **kwargs):
+    if isscalar(expr.dshape):
+        return select(s).scalar()
```

Comment:
@postelrich this is a good start; it'd be better to leverage `odo.into` here, since it can handle more cases.

```python
return odo(s, <scalar dtype from dshape>)
```